website/pages/_app.js

```diff
@@ -38,7 +38,7 @@ export default function App({ Component, pageProps }) {
         <meta name="og:description" property="og:title" content={description} />
       </HashiHead>
       {ALERT_BANNER_ACTIVE && (
-        <AlertBanner {...alertBannerData} product="boundary" />
+        <AlertBanner {...alertBannerData} product="boundary" hideOnMobile />
```

Comment:
Hides alert-banner on mobile, to align with versioned docs needs. This matches previous behaviour - which was previously enabled by globally scoped overrides in `docs-page`.